server/api/observation/observation.router.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+import getObservationController from './observation.controller.get';
+import createObservationController from './observation.controller.create';
+import { requireInternalUser } from '../../auth';
+
+
+const observationRouter = express.Router() as any;
+
+observationRouter.get('/', requireInternalUser, getObservationController.getAllObservations);
+observationRouter.post('/', requireInternalUser, express.json(), createObservationController.createObservation);
```

Comment:
bodyParser.json() is deprecated so it should not be used. 
https://stackoverflow.com/questions/24330014/bodyparser-is-deprecated-express-4

Reply:
It's not technically deprecated according to the accepted answer to that question. It's just bundled into express now. If we're going to use `express.json()`, let's change it across the entire project all at once to avoid confusion. I'm fine with using either. I suppose eliminating the `body-parser` dependency would be nice.

Reply:
I got a depreciation warning that is why I claimed it was deprecated.

Reply:
fixed